docs/src/structuring_element.md

```diff
@@ -146,6 +146,38 @@ single line:
      R = CartesianIndices(A)
 ```
 
+## [Symmetricity](@id concept_symmetric)
+
+Among all the structuring elements, the symmetric ones are used most in practice -- they have better
+properties and thus enables optimized algorithm implementation.
```

Comment:
```suggestion
Among all the structuring elements, the symmetric ones are used most in practice -- they have better
properties and their symmetry enables certain implementation optimizations.
```
What exactly does "better properties" mean?

Reply:
Actually, I was considering the comments https://github.com/JuliaImages/ImageMorphology.jl/pull/82#issuecomment-1132315486 and https://github.com/JuliaImages/ImageMorphology.jl/issues/75#issuecomment-1126734673. I haven't yet dived into the optimization part of the theory so I could be wrong here.

Also, the sollie 2004 book[^1] indicates that disk and its approximations (octagon, diamond) are used very widely because of isotropy.

> in Section 3.7.1 Elementary symmetric
> > Many morphological transformations consist in iterating fundamental operators with the elementary symmetric SE.
>
> in Section 3.7.2 Digital approximations of the disc
> > Owing to their isotropy, discs are very attractive SEs.

[^1]: P. Soille, Morphological Image Analysis. Berlin, Heidelberg: Springer Berlin Heidelberg, 2004. doi: [10.1007/978-3-662-05088-0](https://doi.org/10.1007/978-3-662-05088-0).